homeassistant/components/tessie/strings.json

```
@@ -529,5 +529,35 @@
     "command_failed": {
       "message": "Command failed, {message}"
     }
+  },
+  "services": {
+    "navigation_request": {
+      "name": "Set navigation destination",
+      "description": "Set navigation to a destination.",
+      "fields": {
+        "destination": {
+          "name": "Destination",
+          "description": "The destination to navigate to, e.g., '1600 Amphitheatre Pkwy, Mountain View, CA'."
+        },
+        "locale": {
+          "name": "Locale",
+          "description": "The locale to use for the navigation, e.g., 'en-US'."
+        }
+      }
+    },
+    "navigation_gps_request": {
+      "name": "Set navigation GPS coordinates",
+      "description": "Set navigation to some GPS coordinates.",
```

Matching  the above suggestion for the description:

```suggestion
      "description": "Sends destination GPS coordinates to the onboard navigation system.",
```